packages/main/src/TabContainer.ts

```diff
@@ -274,6 +274,19 @@ class TabContainer extends UI5Element {
 	@property()
 	tabsPlacement: `${TabContainerTabsPlacement}` = "Top";
 
+	/**
+	 * Defines whether no tab will be selected initially.
+	 *
+	 * **Note:** By default, if none of the child tabs have the `selected` property set, the first tab will be automatically selected.
+	 * Setting this property to `true` allows preventing this behavior.
+	 * @default false
+	 * @since 2.9.0
+	 * @public
+	 * @experimental
+	 */
+	@property({ type: Boolean })
+	noDefaultSelection = false;
```

Comment:
I think "auto" is better than "default" in this  case, noAutoSelection, because it works always on every rendering not just once initially, which the word "default" kind of implies. 

As far I can see even not initially, if noDefaultSelection is set, and one removes the selected props of all tabs, one will get no selection.